src/winrm_bootstrap.txt

```diff
@@ -0,0 +1,41 @@
+<powershell>
+# https://www.packer.io/docs/builders/amazon/ebs
+
+# Set administrator password
```

Comment:
Please expand this comment to mention that this is a template that must be processed to populate values like the password..

Reply:
resolved with [c510101](https://github.com/cisagov/windows-server-packer/pull/10/commits/c5101011664ca527b9b9b6743338a8758eeed300)